src/services/item/plugins/etherpad/service.ts

```diff
@@ -151,10 +146,10 @@ export class EtherpadItemService {
 
     const checkedMode = await this.checkMode(mode, member, item);
 
-    if (!item.extra?.etherpad) {
+    if (!isEtherpadItem(item) || !item.extra?.etherpad) {
```

Comment:
Same question about whether type guards are required

Reply:
Yes here they are required, other wise it says that `etherpad` does not exist on type `item.extra`